web/modules/custom/hoeringsportal_audit_log/src/Helpers/ConfigHelper.php

```diff
@@ -89,11 +89,21 @@ public function escapeProviderId(string $input) {
   /**
    * Get route names.
    *
-   * @return array<string, string>
-   *   Array of route names.
+   * @return array<string, string>|null
+   *   Array of route names or NULL.
    */
-  public function getRouteNames() {
-    return $this->moduleConfig->get('logged_route_names');
+  public function getRouteNames() : ?array {
+    return (array) ($this->moduleConfig->get('routes_to_audit')['routes'] ?? NULL);
+  }
+
+  /**
+   * Get url patterns from config.
+   *
+   * @return array<string, string>|null
+   *   Array of url patterns or NULL.
+   */
+  public function getUrlPattern() : ?array {
```

Comment:
This will never return `null` – `(array) null === []`:

```suggestion
   * @return array<string, string>
   *   Array of url patterns.
   */
  public function getUrlPattern() : array {
```